### PR TITLE
Ps 5.7 myr 89 90 91 92

### PR DESCRIPTION
**Pull request by george-lorch**

### CODE REVIEW

File: storage/rocksdb/ha_rocksdb.cc [hunk 1]
```diff
@@ -2921,10 +2852,10 @@ static inline void rocksdb_register_tx(handlerton *const hton, THD *const thd,
                                        Rdb_transaction *const tx) {
   DBUG_ASSERT(tx != nullptr);
 
-  trans_register_ha(thd, FALSE, rocksdb_hton, NULL); // TODO: PFS
+  trans_register_ha(thd, FALSE, rocksdb_hton, nullptr);
```
Comment (laurynas-biveinis) [inline storage/rocksdb/ha_rocksdb.cc]: s/FALSE/false, optionally?

File: storage/rocksdb/ha_rocksdb.cc [hunk 2]
```diff
@@ -2921,10 +2852,10 @@ static inline void rocksdb_register_tx(handlerton *const hton, THD *const thd,
...
   if (my_core::thd_test_options(thd, OPTION_NOT_AUTOCOMMIT | OPTION_BEGIN)) {
     tx->start_stmt();
-    trans_register_ha(thd, TRUE, rocksdb_hton, NULL); // TODO: PFS
+    trans_register_ha(thd, TRUE, rocksdb_hton, nullptr);
```
Comment (laurynas-biveinis) [inline storage/rocksdb/ha_rocksdb.cc]: Likewise

File: storage/rocksdb/rdb_mutex_wrapper.h [hunk 3]
```diff
@@ -75,7 +75,11 @@ class Rdb_cond_var : public rocksdb::TransactionDBCondVar {
   Rdb_cond_var &operator=(const Rdb_cond_var &) = delete;
 
 public:
+#ifdef HAVE_PSI_INTERFACE
+  Rdb_cond_var(PSI_memory_key psi_key);
```
Comment (laurynas-biveinis) [inline storage/rocksdb/rdb_mutex_wrapper.h]: explicit? (as typecasting PSI_memory_key to Rdb_cond_var should be wrong)

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]
Minor optional comments, please decide whether to address here

File: storage/rocksdb/ha_rocksdb.cc (hunk 1 above)
Comment (george-lorch) [reply storage/rocksdb/ha_rocksdb.cc]: Good call, fixing the NULL is enough to warrant fixing the bool at the same time.

File: storage/rocksdb/rdb_mutex_wrapper.h (hunk 3 above)
Comment (george-lorch) [reply storage/rocksdb/rdb_mutex_wrapper.h]: Another nice catch, yes, this should definitely be explicit. I'll make these changes then merge.

**Pull request merged**